sky/server/server.py

```diff
@@ -164,8 +165,36 @@ async def dispatch(self, request: fastapi.Request, call_next):
         return await call_next(request)
 
 
+class CacheControlStaticMiddleware(starlette.middleware.base.BaseHTTPMiddleware
+                                  ):
+    """Middleware to add cache control headers to static files."""
+
+    async def dispatch(self, request: fastapi.Request, call_next):
+        if request.url.path.startswith('/dashboard/_next'):
+            response = await call_next(request)
+            response.headers['Cache-Control'] = 'max-age=86400'
```

Comment:
I kind of feel that this is too long. Maybe 1800 or 3600 is better.

Reply:
OK, update it to 3600.